Overriding default of Managed by Terraform entries (#776)
pulumi/pulumi-aws: CHANGELOG.md

=========

## HEAD (Unreleased)

> * Update default `descriptions` to `Managed by Pulumi`. This may cause a diff for users.

Reviewer: ```suggestion
* Update default `descriptions` for several resources to `Managed by Pulumi`.
```
